packages/access-api/src/config.js

```diff
@@ -105,3 +109,36 @@ export function createAnalyticsEngine() {
     _store: store,
   }
 }
+
+/**
+ * Given a config, return a ucanto Signer object representing the service
+ *
+ * @param {object} config
+ * @param {string} [config.DID] - public identifier of the running service. e.g. a did:key or a did:web
+ * @param {string} config.PRIVATE_KEY - multiformats private key of primary signing key
+ */
+export function configureSigner(config) {
+  const signer = Signer.parse(config.PRIVATE_KEY)
+  if (config.DID) {
+    if (!isDID(config.DID)) {
```

Comment:
I think it may be better to use [existing functionality](https://github.com/web3-storage/ucanto/blob/main/packages/validator/src/schema/did.js) for this, for example this will parse and throw if it's invalid:

```ts
DID.match({ method: 'web' }).from(config.DID)
```

It will also take care of refusing `did:key` which would be invalid in this case.